common/lib/opentelemetry/common/utilities.rb

```diff
@@ -84,7 +84,10 @@ def truncate_attribute_value(value, limit)
       end
 
       def untraced
+        # Unsure how this could be cleanly refactored based on the cop examples.
+        # rubocop:disable Style/ExplicitBlockArgument
         OpenTelemetry::Trace.with_span(OpenTelemetry::Trace.non_recording_span(OpenTelemetry::Trace::SpanContext.new)) { yield }
```

Comment:
Can we disable using an end-of-line comment on line 89 instead of wrapping the line? See https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code

Reply:
Sure thing, I had that first and IIRC that made it so I had to also add the _Layout/LineLength_ rule as well 🤣 
I'll double check!

Reply:
I was wrong! I think I prematurely added the LineLength because I assumed it would need it. Good catch :)